app/src/main/java/com/example/hochi/nextcompanion/NextbikeRentalServiceGenerator.java

```diff
@@ -0,0 +1,33 @@
+package com.example.hochi.nextcompanion;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+
+/*
+ * Retrofit configuration for the  'Nextbike Rental API'
+ */
+public class NextbikeRentalServiceGenerator {
+
+    private static final String BASE_URL =  "https://api.nextbike.net/api/";
+
+    //Enables Logging of all HTTP-Traffic 
+    private static HttpLoggingInterceptor logging = new HttpLoggingInterceptor()
+            .setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
What exactly does this do and what is its reason?

Reply:
Does 
3ffc8f3 resolve this?